src/components/SWActivationCard/SWActivationCard.js

```diff
@@ -370,7 +74,9 @@ class SWActivationCard extends React.Component<Props, State> {
       smartWalletState,
       onButtonPress,
```

Comment:
As far as I've noticed, we're never passing `onButtonPress` to this component anymore, so I guess the prop can be deleted?